core/services/ardupilot_manager/mavlink_proxy/MAVProxy.py

```diff
@@ -37,6 +37,8 @@ def convert_endpoint(endpoint: Endpoint) -> str:
                 master_string = f"{master.place}"
             else:
                 master_string = f"{serial_endpoint_as_input(master)}"
+        if master.connection_type == EndpointType.TCPServer:
+            master_string = f"tcp:{master.place}:{master.argument}"
```

Comment:
Is there a test for it ?

Reply:
There is, but the tests currently only verify if the binary runs, but not it's output.

There are several limitations to the ardupilot-manager tests today. Issue #201 is tracking that and I will open a PR focused on extending and fixing them.